### ISSUE TITLE
Example code throws error

### ISSUE THREAD

**Issue by Dexwell** [Uepaa-AG/react-native-p2pkit#9]
The following example code 

```js
onEnabled: function() {
  console.log('p2pkit is enabled')
  p2pkit.enableProximityRanging()
  p2pkit.startDiscovery('', p2pkit.HIGH_PERFORMANCE) //base64 encoded Data (bytes)
}
```

throws the following error in React Native:

> **JSON value 'HIGH_PERFORMANCE' of type NSString cannot be converted to NSDictionary**
> 
> +[RCTConvert NSDictionary:]
>     RCTConvert.m:55
> __41-[RCTModuleMethod processMethodSignature]_block_invoke.135
> -[RCTModuleMethod invokeWithBridge:module:arguments:]
> facebook::react::invokeInner(RCTBridge*, RCTModuleData*, unsigned int, folly::dynamic const&)
> facebook::react::RCTNativeModule::invoke(unsigned int, folly::dynamic&&, int)::$_0::operator()() const
> invocation function for block in facebook::react::RCTNativeModule::invoke(unsigned int, folly::dynamic&&, int)
> _dispatch_call_block_and_release
> _dispatch_client_callout
> _dispatch_queue_serial_drain
> _dispatch_queue_invoke
> _dispatch_queue_override_invoke
> _dispatch_root_queue_drain
> _dispatch_worker_thread3
> _pthread_wqthread
> start_wqthread

_(Also a tip: replace the example code block opening ` ``` ` with ` ```js ` for syntax highlighting)_

**Comment by EskelCz** [comment]
Have you solved this?

**Comment by Dexwell** [comment]
@EskelCz Nope, see https://github.com/Uepaa-AG/react-native-p2pkit/issues/11#issuecomment-420034882. I advise looking into a different solution.

**Comment by EskelCz** [comment]
From the code it seems pretty clear, there a type mismatch. PPKReactBridge.m expects options parameter as NSDictionary, and index.js startDiscovery passes the variable as a string.
iOS doesn't seem to use the performance string anyway.
Android seems to expect a `discoveryPowerMode` key. I'll try to pass it as an object.

**Comment by EskelCz** [comment]
And it worked. :)
`p2pkit.startDiscovery('', { discoveryPowerMode: 'HIGH_PERFORMANCE' })`
did the trick
